cpp/installer.cc

```diff
@@ -813,6 +814,36 @@ void InstallFullTextBackendCronjobs() {
 }
 
 
+void WaitForElasticsearchReady() {
+     const std::string host("localhost");
+     const std::string base_url("http://" + host + ":9200/");
+     const unsigned MAX_ITERATIONS(5);
+     const unsigned SLEEP_TIME(5 * 1000 * 1000);
+
+     for (unsigned iteration = 1; iteration <= MAX_ITERATIONS; ++iteration) {
+         Downloader downloader(base_url);
+         if (downloader.getResponseCode() == 200)
+             break;
+         ::usleep(SLEEP_TIME);
```

Comment:
Why not use sleep instead.  Using usleep signals to the reader that you care about microsecond resolution!